ansible/roles/nginx-proxy/templates/openshift/nginx-configuration.yml

```diff
@@ -20,21 +20,33 @@ data:
 
     http {
 
-      log_format compression '$remote_addr - $remote_user [$time_local] '
-                           '"$request" $status $body_bytes_sent '
-                           '"$http_referer" "$http_user_agent" "$gzip_ratio"';
+      log_format logger-json escape=json '{"source": "nginx", "time": $msec, \
```

Comment:
Is the source "nginx"? Or should this be "operator-pipeline" and "nginx" the name?